org.adempiere.base/src/org/adempiere/util/IProcessUI.java

```diff
@@ -117,6 +117,7 @@ default public void askForInput(String message, MLookup lookup, int displayType,
 
 	/**
 	 * add to list of file available for download after process end 
+	 * think about to use {@link SvrProcess#addDownloadFile(File)} 
```

Comment:
hi @hieplq, it would be good to perform a readability check on the comments added to code (for e.g through bard or chatgpt) to make sure it is easily understandable by others. also, if you do a mouse over check in Eclipse for the javadoc popup (that's what your API user will usually see), the javadoc above is probably not in the easily readable category.

Reply:
thanks for suggest it's helpful
i ask him by this one
this's good english sentence for clarity and readability "my bad english"